letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -643,6 +656,16 @@ def _update_ssl_vhosts_addrs(self, vh_path):
 
         return ssl_addrs
 
+    def _remove_existing_ssl_directives(self, vh_path):
+        for directive in ["SSLCertificateKeyFile", "SSLCertificateChainFile",
+                          "SSLCACertificatePath", "SSLCertificateFile"]:
```

Comment:
I don't think we shouldn't remove `SSLCACertificatePath`; that is used for authenticating client certificates.